types/snarkjs/index.d.cts

```diff
@@ -0,0 +1,118 @@
+export as namespace snarkjs;
+
```

Comment:
this specific one denotes and is being used for global scripts (loaded via script) within a browser. Is that required?

Reply:
I think it could, there should be an `iife` dist -> https://github.com/iden3/snarkjs/blob/master/config/rollup.iife.config.js